tests/structuredheadings/formatcombo.js

```diff
@@ -33,23 +38,50 @@
       });
     },
 
-    "if previous heading is autonumber, the new heading is autonumbered": function () {
+    "increments autonumbering levels and preset": function () {
```

Comment:
Preset should probably be something like numberingScheme now that I think about it